synapse/rest/admin/media.py

```diff
@@ -109,6 +109,28 @@ async def on_POST(self, request, server_name: str, media_id: str):
         return 200, {}
 
 
+class ProtectMediaByID(RestServlet):
+    """Protect local media from being quarantined.
+    """
+
+    PATTERNS = admin_patterns("/media/protect/(?P<media_id>[^/]+)")
+
+    def __init__(self, hs):
```

Comment:
Should we add type hints while we're here (otherwise it might conflict with https://github.com/matrix-org/synapse/pull/9093?)

Reply:
Yeah, we should...although #9093 doesn't touch anything in the admin dir. 😄

Reply:
Oh yes this is in the admin dir not media! :facepalm: